### DISCUSSION TITLE
Seeking Guidance on Contributing

### DISCUSSION THREAD

**Discussion by RichieOrito** [camicroscope/caMicroscope#835] [General]
I hope you're all doing well. I'm reaching out to seek some guidance on how to get started with contributing to the caMicroscope project. As I'm eager to contribute and learn, I'm having trouble identifying specific issues to work on since they're not readily provided.
 
Could someone please provide some direction on how I can begin contributing? Whether it's tackling existing issues, working on new features, or any other area where help is needed, I'm open to suggestions. @birm
 
 Looking forward to contributing to the caMicroscope project!

**Comment by gbengaoluwadahunsi** [comment]
Hello @RichieOrito  and welcome
here is a link to a guide on what the project entails and how you can contribute,
https://github.com/orgs/camicroscope/discussions/686

cheers.
